samples/tests/GPIOOutputs-k64f.js

```diff
@@ -0,0 +1,43 @@
+// Copyright (c) 2017, Intel Corporation.
+
+// Test code for FRDM-K64F that toggles each of pins D1-D15 as outputs
+// indefinitely. Configures D0 as an input and prints a message whenever it
+// changes, so you connect a jumper wire from D0 to each of the other pins (one
+// at a time) and see which ones generate a console message.
+
+console.log("Test external GPIOs as outputs...");
+
+console.log("\nWire D0 to each of D1-D15 in turn!");
```

Comment:
Yeah, but I think this is better - this is an 'extra' newline for a blank line; console.log already puts a newline. If I put it at the end it would look more like a normal newline.